includes/Google/u2f-api.js

```diff
@@ -0,0 +1,361 @@
+// Copyright 2014 Google Inc. All rights reserved
+//
+// Use of this source code is governed by a BSD-style
```

Comment:
We'll need to read the source of the license and confirm that it's compatible to use in a GPL project such as WordPress (I think we can, just want to confirm)